test/screens/play/view_active_quests_page_test.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:treasure_of_the_high_seas/screens/play/player_hand.dart';
+import 'package:treasure_of_the_high_seas/screens/play/view_active_quests_page.dart';
+
+import '../../mocks.dart';
+import '../../test_utils.dart';
+
+void main() {
+  testWidgets('close button should return to play page', (WidgetTester tester) async {
+    final audioModel = MockAudioModel();
+    await launchGameFromMenuMock(tester, audioModel: audioModel);
+
+    await tester.pumpAndSettle();
+
+    final activeQuestsFinder = find.byIcon(Icons.assignment);
+    expect(activeQuestsFinder, findsOneWidget);
+    await tester.tap(activeQuestsFinder);
+    await tester.pumpAndSettle();
+
+    expect(find.byType(ViewActiveQuestsPage), findsOneWidget);
+    expect(find.byType(PlayerHand), findsNothing);
+
+    final exitFinder = find.text('Close');
+    expect(exitFinder, findsOneWidget);
```

Comment:
We can probably do away with these `findsOneWidget` assertions, since it's built into what happens when we call `tap`. In general you only need to do this if you're not planning to interact with the component and just want to prove it's on the screen somewhere.

Reply:
Nice, I've removed them. This test was heavily inspired by the existing view_discard_page_test, where there are more of these redundant assertions.